cpp/data/installer/sudo.zts-restart

```diff
@@ -0,0 +1,4 @@
+# Allow apache to restart the zts service
+Cmnd_Alias ZTS_RESTART = /bin/systemctl restart zts
+Cmnd_Alias ZTS_STATUS = /bin/systemctl status zts
+vufind  ALL = NOPASSWD: ZTS_RESTART, ZTS_STATUS
```

Comment:
Apache will only run as user `vufind` if we have a vufind installation. On ub-tools-only under CentOS the username is `apache`, if we might have any constellation where ub-tools-only is installed under Ubuntu it might also be "www-data".

Reply:
Additional potential users have been included.